### PR TITLE
One line navbar

### LINKED ISSUE #517: NavBar goes on two lines on small screens
Need to make it so that the navbar portion that keeps stretching onto two lines stays on one and uses the available space.

This is due to the last-updated-indicator wrapping

### PR DESCRIPTION
**Pull request by ryan-conn**
## Description
Makes the navbar stay on one line rather than become thicc af

## Rationale

It starts looking a bit weird when the width goes below 600px (term select clips with name):
image
but this should be fixed with the mobile functionality PR and is incredibly unlikely on desktop so I don't think it's that big a deal

## How to test

Describe the best way to test the specific changes you made. If it's covering a special case,
make sure to include the situations they should check out (i.e. PHYS 207 - 501, term 202031)

## Screenshots
lol yikes

|Before|After|
|--|--|
| image | image |

## Related tasks
Closes #517.

### CODE REVIEW

File: autoscheduler/frontend/src/components/NavBar/NavBar.css [hunk 1]
```diff
@@ -1,4 +1,9 @@
-.name-and-button {
+.logout-container {
     display: flex;
     align-items: center;
+    min-width: 0;
```
Comment (firejake308) [inline autoscheduler/frontend/src/components/NavBar/NavBar.css:4]: Why is this necessary?

File: autoscheduler/frontend/src/components/LastUpdatedAt.tsx [hunk 2]
```diff
@@ -110,17 +110,17 @@ const LastUpdatedAt: React.FC = () => {
...
   return (
-    <div
+    <Typography
       style={{
-        display: 'flex',
-        alignItems: 'center',
         paddingLeft: 8,
+        whiteSpace: 'nowrap',
+        overflow: 'hidden',
+        textOverflow: 'ellipsis',
```
Comment (firejake308) [inline autoscheduler/frontend/src/components/LastUpdatedAt.tsx:118]: We should probably be doing CSS class for this, but since this is the only thing in the component that would need it, I think it's not worth creating a new CSS file just for this

**Review by firejake308 — COMMENTED** [review]
Not sure I understand what min-width is doing, but it works great on my small computer running Chrome

File: autoscheduler/frontend/src/components/NavBar/NavBar.css (hunk 1 above)
Comment (ryan-conn) [reply autoscheduler/frontend/src/components/NavBar/NavBar.css:4]: It's here because the text for login/logout used to shrink just like the last updated text, but I forgot to remove this line when I changed it

File: autoscheduler/frontend/src/components/LastUpdatedAt.tsx (hunk 2 above)
Comment (ryan-conn) [reply autoscheduler/frontend/src/components/LastUpdatedAt.tsx:118]: Yeah that was my thought as well, but I kept it as a style since that's how it was originally (which I think is fine until we need multiple of them)

**Comment by firejake308** [comment]
The latest commit looks fine on all screens except iPhone 5/SE

localhost_8000_schedule(iPhone 5_SE) (1)

**Review by firejake308 — DISMISSED** [review]
Works on every screen that I tried except for iPhone 5/SE, which I will handle in #544
